bin/yg

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# create $HOME/npmbin directory and set that in your PATH
+# export PATH="$HOME/npmbin/node_modules/.bin:$PATH"
+# https://github.com/Schniz/fnm/issues/110
+yg() {
```

Comment:
Could you provide some context on this? Is this a command I should run manually?

Reply:
instead of having a variety of node modules mixed throughout different node versions, you can set up an npm bin that'll install them under one version every time, no matter what version of node your project uses!

so instead of `yarn global add expo-cli` you'd do `yg expo-cli` 
you can call this alias whatever you'd like though :smile: